test/Quake/complex.qke

```diff
@@ -0,0 +1,46 @@
+// ========================================================================== //
```

Comment:
Do we still need const-prop-complex pass? Maybe we can just replace it with the canonicalizer pass when needed?

Reply:
I left it for now, but we might be able to replace it (soon?).